modules/video_writer.py

```diff
@@ -0,0 +1,76 @@
+import cv2
+import numpy as np
+
+
+class VideoWriter:
+
+    def __init__(self, outfile=None):
+        self.outfile = outfile
+        self.frame_resolution = None
+        self.fps = 30  # TODO: read from video file eventually
```

Comment:
Suggest to get fps and frame resolution from `VideoReader`. `VideoCapture` provides fps and you can get frame resolution by reading 1 frame in `VideoReader` init, then rewind `VideoCapture`. So, save fps and resolution in `VideoReader` init and pass it to this init (it will not be lazy in this case).